src/repositories/live.repository.ts

```diff
@@ -0,0 +1,330 @@
+
+  public async createQuestion(broadcastId: string, question: string): Promise<any> {
+    // TODO: not enabled?
+    const { body } = await this.client.request.send({
+      url: `/api/v1/live/${broadcastId}/questions/`,
+      method: 'POST',
+      form: {
+        _csrftoken: this.client.state.cookieCsrfToken,
+        _uuid: this.client.state.uuid,
+        text: question,
+      },
+    });
+    return body;
+  }
+
+  public async showQuestion(broadcastId: string, questionId: string): Promise<LiveShowQuestionResponseRootObject> {
+    // TODO: not working on client
+    const { body } = await this.client.request.send<LiveShowQuestionResponseRootObject>({
+      url: `/api/v1/live/${broadcastId}/question/${questionId}/activate/`,
+      method: 'POST',
+      form: {
+        _csrftoken: this.client.state.cookieCsrfToken,
+        _uuid: this.client.state.uuid,
+      },
+    });
+    return body;
+  }
+
+  public async hideQuestion(broadcastId: string, questionId: string): Promise<LiveHideQuestionResponseRootObject> {
+    const { body } = await this.client.request.send<LiveHideQuestionResponseRootObject>({
+      url: `/api/v1/live/${broadcastId}/question/${questionId}/deactivate/`,
+      method: 'POST',
+      form: {
+        _csrftoken: this.client.state.cookieCsrfToken,
+        _uuid: this.client.state.uuid,
+      },
+    });
+    return body;
+  }
+
+  public async getQuestions(): Promise<LiveGetQuestionsResponseRootObject> {
+    const { body } = await this.client.request.send<LiveGetQuestionsResponseRootObject>({
+      url: '/api/v1/live/get_questions/',
+      method: 'GET',
+    });
+    return body;
+  }
+
+  public async wave(broadcastId: string, viewerId: string): Promise<LiveWaveResponseRootObject> {
+    const { body } = await this.client.request.send<LiveWaveResponseRootObject>({
+      url: `/api/v1/live/${broadcastId}/wave/`,
+      method: 'POST',
+      form: this.client.request.sign({
+        viewer_id: viewerId,
+        _csrftoken: this.client.state.cookieCsrfToken,
+        _uid: this.client.state.cookieUserId,
+        _uuid: this.client.state.uuid,
+      }),
+    });
+    return body;
+  }
+
+  public async like(broadcastId: string, likeCount: number = 1): Promise<LiveLikeResponseRootObject> {
+    const { body } = await this.client.request.send<LiveLikeResponseRootObject>({
+      url: `/api/v1/live/${broadcastId}/like/`,
+      method: 'POST',
+      form: this.client.request.sign({
+        _csrftoken: this.client.state.cookieCsrfToken,
+        _uid: this.client.state.cookieUserId,
+        _uuid: this.client.state.uuid,
+        user_like_count: likeCount,
+      }),
+    });
+    return body;
+  }
+
+  public async getLikeCount(broadcastId: string, likeTs: string = ''): Promise<LiveLikeCountResponseRootObject> {
+    const { body } = await this.client.request.send<LiveLikeCountResponseRootObject>({
+      url: `/api/v1/live/${broadcastId}/get_like_count/` + (likeTs === '' ? '' : `?like_ts=${likeTs}`),
```

Comment:
Please, refactor all the places with template strings in `url` parameter to `qs` parameter.